paper.md

```diff
@@ -0,0 +1,83 @@
+- Grid power can be treated as a primary or backup source of electricity with a user-defined availability profile.
+These electricity sources can be included as the single source of electricity or in any combination as part of a hybrid system.
+
+## Electricity demand
+Users can generate electricity load profiles based on appliance ownership, usage patterns, power demands, and other device-specific information. Load profiles are based on the probability that individual user-specified appliances are in use, which can vary throughout the day and year, using processes described in `@Sandwell:2016` and applied in `@Chambon:2020`. Any kind of electrical appliance and usage pattern can be defined by the user and ownership of appliances can be set to increase over time. Demands can be segregated into `domestic`, `commercial`, `public`, or other demand categories and included independently within the system. Alternatively users can provide their own hourly load profile, for example taken from monitored data.
+
+## Simulation
+CLOVER assesses the balance of energy sources and demands at an hourly resolution over a user-specified lifetime period. During this process data, including energy flows, battery state of charge, demand satisfaction, and times at which electricity is (un)available, are recorded.
+
+The simulation functionality of CLOVER can be used to explore the technical performance of existing or theoretical case study systems. Data from the simulation process can be processed to give insight into how a system might perform over an average day, year, or over its lifetime. These could be used explore opportunities to maximise electricity utilisation by integrating additional user loads or investigating the impact of incremental increases in storage capacity for more efficient system design `[@Sandwell:2017b]`.
+
+## Optimisation
+CLOVER can identify the optimum energy system, defined by a selected optimisation criterion, subject to any number of user-specified constraints (threshold criteria). Potential optimisation criteria include lowest cost, lowest greenhouse gas emissions, or others depending on the goals of the system; potential threshold criteria could set minimum allowable levels for the reliability of electricity supply, energy demand satisfaction, renewable energy penetration, or many more.
+
+Users can specify the types of technologies available to include in the system and CLOVER uses an iterative heuristic search algorithm to identify the optimum combination and capacities to meet the needs of the community over its lifetime. A diagrammatic overview of this process is shown in @Beath:2021. The optimisation process has been used to explore the economic and greenhouse gas emissions impact of off-grid and grid-connected minigrids in India `[@Sandwell:2017a]` and the potential to offset diesel generation using solar energy in refugee camps in Rwanda `[@BarandaAlonso:2021]`.
```

Comment:
Looks like we don't need backticks, i.e., this reference "@Beath:2021" is correct and the ones in backticks aren't showing up.

See attached pdf: [paper.pdf](https://github.com/CLOVER-energy/CLOVER/files/9423151/paper.pdf)

Reply:
I've removed the backticks from the references - does it work now?